tools/src/icon4py/tools/py2fgen/_utils.py

```diff
@@ -48,6 +53,7 @@ def setup_logger(name: str, log_level: int = logging.INFO) -> logging.Logger:
     return logger
 
 
-def parse_comma_separated_list(ctx: click.Context, param: click.Parameter, value: str) -> list[str]:
+def parse_comma_separated_list(_: click.Context, __: click.Parameter, value: str) -> list[str]:
```

Comment:
Just an optional suggestion: if args are meant to be used as positional only, making them explicitly so could be cleaner...
```suggestion
def parse_comma_separated_list(_: click.Context, __: click.Parameter, value: str, /) -> list[str]:
```